hexrd/ui/interactive_template.py

```diff
@@ -12,6 +12,7 @@ def __init__(self, img, parent=None):
         self.parent = parent.image_tab_widget.image_canvases[0]
         self.ax = self.parent.axes_images[0]
         self.raw_axes = self.parent.raw_axes[0]
+        self.panels = self.parent.iviewer.instr._detectors
```

Comment:
This seems to be reaching into a private member, is there a better way we can access the detectors? If we are going to use a private member, we should make it public ...

Reply:
Yes! There is a public property that can be accessed, this has been fixed.

Reply:
was about to say...  Thanks!